api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -54,6 +54,7 @@
   private static final String RAB_SUPPORT_EMAIL = "aouresourceaccess@od.nih.gov";
 
   private static final String WELCOME_RESOURCE = "emails/welcomeemail/content.html";
+  private static final String WELCOME_RESOURCE_OLD = "emails/welcomeemail/content_existing.html";
```

Comment:
I'd prefer to be consistent with the names, so if we have "old" then I'd also like to have `content_old.html` and `sendOldWelcomeEmail()`